### PR TITLE
Convert week_number.jsx to flow (as a simple example)

### PR DESCRIPTION
**Pull request by aij**
This is a pretty minimal conversion. We could make `onClick` and `handleClick` be functions that take a `SyntheticEvent` for stricter checking.

### CODE REVIEW

File: src/week_number.jsx [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
 
-export default class WeekNumber extends React.Component {
+export default class WeekNumber extends React.Component<{
+  weekNumber: number,
+  onClick?: Function
+}> {
   static propTypes = {
     weekNumber: PropTypes.number.isRequired,
     onClick: PropTypes.func
   };
```
Comment (aij) [inline src/week_number.jsx]: If we were to use babel-plugin-flow-react-proptypes, these `propTypes` could be removed as they would be automatically generated from the types above.

**Comment by martijnrusschen** [comment]
Nice work 👍

**Comment by martijnrusschen** [comment]
Looks like the CI integration works, shall we go ahead and merge this?

**Comment by aij** [comment]
@martijnrusschen Sure, this should be fine to merge.

I was toying around with webpack integration, but unsure where it would be best. `server.js` and `webpack.docs.config.js` are two options. Is `webpack.config.js` actually used anywhere?

**Comment by martijnrusschen** [comment]
Good question, need to dive into that. I was planning on upgrading webpack as well, but haven't got the time to make this happen.

**Pull request merged**